src/components/Header.tsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from 'react';
+
+const Header = () => {
+  const [userName, setUserName] = useState('Juan Pérez');
+
+  return (
+    <div className="flex justify-end h-[96px] w-full bg-[#1E1E1E]">
```

Comment:
1. let's try to avoid using custom hex colours. I already defined them in the tailwind config file, so please change this to bg-light.
2. Use rem or em units, not px. 
3. set a margin, instead of a fixed height

Reply:
[1, 2] Done
[3] I always used heights, could you help me with that? How can I use margins in this case?